platforms/core-execution/persistent-cache/src/test/groovy/org/gradle/cache/internal/DefaultCacheBuilderTest.groovy

```diff
@@ -26,19 +26,19 @@ import static org.gradle.cache.FileLockManager.LockMode.OnDemand
 import static org.gradle.cache.FileLockManager.LockMode.Shared
 import static org.gradle.cache.internal.filelock.DefaultLockOptions.mode
 
-class DefaultUnscopedCacheBuilderFactoryTest extends Specification {
+class DefaultCacheBuilderTest extends Specification {
```

Comment:
💭 `UnscopedCacheBuilderFactory` is just a thin service wrapper of the only meaningful implementation of `CacheBuilder`. Now we test the meaningful thing.